constants/megaMenuLinks.ts

```diff
@@ -483,45 +483,52 @@ const QUANTUM_COMPUTING_LABS : MegaDropdownMenuGroup = {
       }
     },
     {
-      label: 'Lab 2. Quantum Measurement',
-      url: `${baseUrl}${pathLabs}/lab-2-quantum-measurements`,
+      label: 'Lab 2. Single Quibit Gates',
+      url: `${baseUrl}${pathLabs}/lab-2-single-quibit-gates`,
       segment: {
-        cta: 'lab-2-quantum-measurements', location: sectionLabs
+        cta: 'lab-2-single-quibit-gates', location: sectionLabs
```

Comment:
Typo, fixable after https://github.com/qiskit-community/platypus/pull/220 is merged:

```suggestion
        cta: 'lab-2-single-qubit-gates', location: sectionLabs
```